src/snowflake/snowpark/session.py

```diff
@@ -498,6 +497,9 @@ def __init__(
         self._tmpdir_handler: Optional[tempfile.TemporaryDirectory] = None
         self._runtime_version_from_requirement: str = None
 
+        # local import due to pex issue.
```

Comment:
```suggestion
        # Local import due to pex issue.
```

Reply:
I guess this answers my question. Maybe a TODO to eventually fix this?

Reply:
added a comment.